python/Ganga/GPIDev/Base/Objects.py

```diff
@@ -35,10 +35,11 @@ def _getName(obj):
     if returnable is None:
         returnable = getattr(getattr(obj, '__class__', None), '__name__', None)
     if returnable is None:
-        returnable = str(obj)
-    return returnable
+        return str(obj)
+    else:
+        return returnable
```

Comment:
There is a general principle to prefer single exit points for functions. It's not a hard rule but it does make understanding the flow of a function easier. Given that this change is functionally equivalent to the previous version but is more complex, is it worth it?